### ISSUE TITLE
Little bug when i download anime

### ISSUE THREAD

**Issue by Dokman** [einstein95/crunchy-xml-decoder#82]
![image](https://cloud.githubusercontent.com/assets/2290904/18735196/504fcccc-807c-11e6-93c1-dd64ac710903.png)
it's just formatting but well it's better with a good characters

**Comment by jsonn** [comment]
It's using an ANSI escape sequence to clear the current line. I have no idea what is supported by cmd.exe for that purpose.

**Comment by darkhelmet2016** [comment]
Why are the output files smaller then usual for videos?

**Comment by jsonn** [comment]
HLS is using VBR and maybe even a lower effective bitrate for the same resolution.

**Comment by Kamekameha** [comment]
No, the RTMP streams were already VBR, more specifically, using the 2 pass ratecontrol. Their HLS ones are encoded using CRF, which it's technically better but for some reason Crunchyroll decided to use worst overall, and not even consistent, x264 settings than their RTMP counterparts, thus making them not only even smaller (their x264 settings on the RTMP streams are already bad tbh) but also worse in quality terms. So yeah, technically "better", no more than that. Anyway, that shouldn't really matter for most of their shows as this is mostly noticeable on scenes with high amount of action.

Also, you could look up for the x264 options used on either the HLS or RTMP stream's SEI, it really worth a shot to read if you really want to say that either stream is VBR or not, when, in some way or another, both are.

**Comment by albogango** [comment]
Has this issue yet been solved? I'm experiencing the same thing when downloading 1080p. If so, is there a layman's solution?

**Comment by jaw20** [comment]
@Kamekameha Using media info, (i think this is what you are looking for) I grabbed the encoding setting for a ts file that was being downloaded. ``cabac=1 / ref=4 / deblock=1:1:1 / analyse=0x3:0x113 / me=hex / subme=6 / psy=1 / psy_rd=0.40:0.00 / mixed_ref=1 / me_range=16 / chroma_me=1 / trellis=1 / 8x8dct=1 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=-2 / threads=1 / lookahead_threads=1 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=5 / b_pyramid=2 / b_adapt=1 / b_bias=0 / direct=1 / weightb=1 / open_gop=0 / weightp=1 / keyint=48 / keyint_min=24 / scenecut=40 / intra_refresh=0 / rc_lookahead=30 / rc=crf / mbtree=1 / crf=25.7 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / vbv_maxrate=8000 / vbv_bufsize=12000 / crf_max=0.0 / nal_hrd=none / filler=0 / ip_ratio=1.40 / aq=1:0.60``

and here is the mediainfo file.
[hls.txt](https://github.com/einstein95/crunchy-xml-decoder/files/627162/hls.txt)

**Comment by Kamekameha** [comment]
@jaw20 sorry, I'm kind of missing your point there. I based what I said on my previous comment off of both the RTMP and HLS streams' SEI (i.e. encoding settings like the one you just posted), so you're basically proving my point, unless you meant something else of course.
